types/bootbox/index.d.ts

```diff
@@ -7,9 +7,9 @@
 /// <reference types="jquery" />
 
 /** Bootbox options shared by all modal types */
-interface BootboxBaseOptions {
+interface BootboxBaseOptions<T = any> {
 	title?: string | Element;
-	callback?: (result: boolean | string) => any;
+        callback?: (result: T) => any;
```

Comment:
Nit: fix the indentation here.

Reply:
Huh, I didn't think anybody used tabs in DT. Fixed.